cluster/addons/iso-client/README.md

```diff
@@ -0,0 +1,95 @@
+### Cgroup-cpuset-cpus isolator.
+
+This is an example implementation of extended pod-level resource isolation as specified [here](https://github.com/kubernetes/features/issues/246).
+Cgroup-cpuset-cpus isolator aims to provide a way for an operator to specify which PODs should be run with exclusive [cpusets](http://man7.org/linux/man-pages/man7/cpuset.7.html).
+
+### Building 
+
+In order to use this feature, one has to build a modified kubelet which resides in Intel's fork of Kubernetes [here](https://github.com/intelsdi-x/kubernetes/tree/ext-iso).
+
+First issue:
```

Comment:
I think that it will be easier for an user, to just follow our commands then to read k8s documentation how to regenerate protobuf and compile kubelet.